config/jobs/gardener/gardener-e2e-kind-ha-multi-zone.yaml

```diff
@@ -0,0 +1,79 @@
+presubmits:
+  gardener/gardener:
+  - name: pull-gardener-e2e-kind-ha-multi-zone
+    cluster: gardener-prow-build
+    always_run: true
```

Comment:
Please add an `optional: true` flag to this job for the beginning. It will run on each PR, but its success won't be enforced by branch protection rules. When we know, that it is stable, we can remove the flag.